pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -43,6 +43,28 @@ configmap/nvidia-feature-discovery-preprovisioned-example-rhel-84 created
 dkp create cluster preprovisioned --cluster-name ${CLUSTER_NAME} --control-plane-endpoint-host <control plane endpoint host> --control-plane-endpoint-port <control plane endpoint port, if different than 6443> --override-secret-name=$CLUSTER_NAME-user-overrides
 ```
 
+<p class="message--note"><strong>NOTE: </strong>If your cluster is air-gapped or you have a local docker registry you must provide additional arguments when creating the cluster. </p>
+
+```bash
+export DOCKER_REGISTRY_ADDRESS=<https/http>://<registry-address>:<registry-port>
+export DOCKER_REGISTRY_CA=<path to the CA on the bastion>
+```
+
+- `DOCKER_REGISTRY_ADDRESS`: the address of an existing Docker registry accessible in the VPC that the new cluster nodes will be configured to use a mirror registry when pulling images.
+- `DOCKER_REGISTRY_CA`: (optional) the path on the bastion machine to the Docker registry CA. Konvoy will configure the cluster nodes to trust this CA. This value is only needed if the registry is using a self-signed certificate and the AMIs are not already configured to trust this CA.
+- `DOCKER_REGISTRY_USERNAME`: optional, set to a user that has pull access to this registry.
+- `DOCKER_REGISTRY_PASSWORD`: optional if username is not set.
+
+```bash
+dkp create cluster preprovisioned --cluster-name ${CLUSTER_NAME} \
```

Comment:
If you're using a registry with a path, such as https://harbor-registry.daclusta/harbor-registry, you will need to configure it differently as this does not properly pull 
@dkoshkin

Reply:
I believe this is fixed in rc11, right @dkoshkin ?